Mathlib/Data/Dfinsupp/Order.lean

```diff
@@ -17,13 +17,19 @@ This file lifts order structures on the `α i` to `Π₀ i, α i`.
 
 ## Main declarations
 
-* `dfinsupp.order_embedding_to_fun`: The order embedding from finitely supported dependent functions
+* `Dfinsupp.orderEmbeddingToFun`: The order embedding from finitely supported dependent functions
   to functions.
 
 -/
 
+-- porting note: The unusedVariables linter has false positives in code like
+--    variable (α) [∀ i, Zero (α i)]
+-- marking the  `i` as unused. So disable that lint for now.
+-- Likely issue https://github.com/leanprover/lean4/issues/2088, so remove when that is fixed
+set_option linter.unusedVariables false
 
-open BigOperators
+-- Porting note: Unkonwn namespace BigOperators
+-- open BigOperators
```

Comment:
Please merge master and re-enable this locale/namespace.

Reply:
Even if it is not used? 
Or did you change things in the file because `BigOperators` was not there, @int-y1 ?

Reply:
Re-enabled. I didn't notice anything related to `BigOperators` while fixing the file.

> Even if it is not used?

For now, it's probably fine to keep `BigOperators`.